src/MongoDB.Driver.Core/Core/Operations/ParallelScanOperation.cs

```diff
@@ -52,6 +52,7 @@ public class ParallelScanOperation<TDocument> : IReadOperation<IReadOnlyList<IAs
         /// <param name="numberOfCursors">The number of cursors.</param>
         /// <param name="serializer">The serializer.</param>
         /// <param name="messageEncoderSettings">The message encoder settings.</param>
+        [Obsolete("ParallelScanOperation was deprecated in version 4.1.")]
```

Comment:
Using the `Obsolete` attribute is exactly what we need 👍  I think due to ambiguity between driver versions and server versions, it would probably be more precise to specify that the operation was "deprecated in server version 4.1."